spec/timed-round-robin_spec.rb

```diff
@@ -58,6 +58,15 @@
         worker.process
       end
     end
+
+    it "will check for new queues until it has some" do
+      worker = Resque::Worker.new('*')
+      worker.process
+      5.times { Resque::Job.create(:q1, SomeJob) }
```

Comment:
If you stuck a `Timecop.travel(Time.now + 60)` here around the work on line 66, wouldn't this work with the original logic?

It seems like we're special casing the the empty queue situation to be acted on immediately, but moving from 1 -> 2 queues (or any number > 0) will continue to wait 60 seconds. Does it need to be special cased like that?